### ISSUE TITLE
Feature request: Add tags to searchable content, so that sword finds also tags

### ISSUE THREAD

**Issue by chrisi51** [tpwd/ke_search#183]
We just recognized, that the `sword` from the normal searchbox does not seem to be used to search in the tags of the index.

So if i have a tag `foo` on a news for example, it is listed in the `tags` column of `tx_ke_search_index` of the corresponding entry. But if i just put `foo` in the searchbar, it is not getting found. if i put `foo` at the end of the `content` column of the entry, it is found.

So it seems, you can't search for tags, except if you put in a corresponding filter? Or do i have any configuration issue or is it even a bug?

Our tags are very dynamic and so it would be not practicable to put all those tags into single filter.

Thx for any help =)

**Comment by Inovanet** [comment]
Is there something new to this feature?

**Comment by christianbltr** [comment]
No, not yet. Are interesting in sponsoring this feature or providing a Pull Request?

**Comment by christianbltr** [comment]
This feature has been added in master and will be in version 6.4.0.
